staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -754,3 +760,27 @@ func Warning(cmdErr io.Writer, newGeneratorName, oldGeneratorName string) {
 		oldGeneratorName,
 	)
 }
+
+// ParseLabels turns a string representation of a label set into a map[string]string
+func ParseLabels(labelSpec interface{}) (map[string]string, error) {
+	labelString, isString := labelSpec.(string)
+	if !isString {
+		return nil, fmt.Errorf("expected string, found %v", labelSpec)
+	}
```

Comment:
I know this function has been migrated from generators but...

If we always expect a string to happen here, is there any reason not changing this function signature to receive a string, and remove the interface validation? This is going to be used gradually on functions, and to parse labels, so I would change this not to be interface and simplify things :)